projects/odf_transform/geojson_files/Azmp_pmza_transects.geojson

```diff
@@ -0,0 +1,51 @@
+{
+  "type": "FeatureCollection",
+  "name": "AZMP DFO \/ PMZA MPO",

```

Comment:
can you merge 'name' into 'section name'  to create an 'name' propoerty for each polygon ?